packages/manager/apps/telecom/src/app/telecom/telephony/billingAccount/guides/telecom-telephony-guides.controller.js

```diff
@@ -1,43 +1,50 @@
-import flatten from 'lodash/flatten';
-import map from 'lodash/map';
+import { GUIDES } from './telecom-telephony-guides.constants';
 
-import { Environment } from '@ovh-ux/manager-config';
+export default class TelecomTelephonyGuidesCtrl {
+  /* @ngInject */
+  constructor(SessionService) {
+    this.SessionService = SessionService;
+  }
 
-angular.module('managerApp').controller(
-  'TelecomTelephonyGuidesCtrl',
-  class TelecomTelephonyGuidesCtrl {
-    constructor(TELEPHONY_GUIDES) {
-      this.constant = { TELEPHONY_GUIDES };
-    }
+  $onInit() {
+    this.loading = {
+      init: true,
+    };
+    this.guides = null;
 
-    $onInit() {
-      this.loading = {
-        init: false,
+    this.guides = { sections: [] };
+
+    this.SessionService.getUser().then(({ ovhSubsidiary }) => {
+      this.language = ovhSubsidiary;
+
+      this.guides = {
```

Comment:
I pre-build the guides list object based on `ovhSubsidiary` instead of filtering link in template.